rosagent.py

```diff
@@ -31,8 +31,16 @@ def __init__(self):
         topic = "/{}/camera_node/camera_info".format(self.vehicle)
         self.cam_info_pub = rospy.Publisher(topic, CameraInfo, queue_size=1)
 
+
+
+
         # copied from camera driver:
 
+        left_encoder_topic = "/{}/left_wheel_encoder_node/tick".format(self.vehicle)
+        self.left_encoder_pub = rospy.Publisher(left_encoder_topic, WheelEncoderStamped, queue_size=1)
+        right_encoder_topic = "/{}/right_wheel_encoder_node/tick".format(self.vehicle)
+        self.right_encoder_pub = rospy.Publisher(right_encoder_topic, WheelEncoderStamped, queue_size=1)
```

Comment:
Any reason why we are using two topics instead of having WheelEncoderStamped contain both wheels ?

Reply:
this is the way the interface is in dt-duckiebot-interface so I matched that as much as possible. On the real hardware I think there's no requirement that both encoders fire at the same time.